dotnet/src/Extensions/PromptTemplates.Prompty/Extensions/PromptyKernelExtension.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticKernel.Experimental.Prompty.Extension;
+
+public static class PromptyKernelExtension
+{
+    public static KernelFunction CreateFunctionFromPrompty(
+        this Kernel _,
+        string promptyPath)
+    {
+        var prompty = new Core.Prompty();
+        prompty = prompty.Load(promptyPath, prompty);
+        var promptFunction = new PromptyKernelFunction(prompty);
```

Comment:
Does this need to be a custom KernelFunction-derived type? I was expecting it to instead be something based on implementing IPromptTemplateFactory. @markwallace-microsoft, what would you recommend?

Reply:
Nope